src/ragger/conftest/configuration.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, Any
+
+# The following parameters, if any, are required.
+# Please set their values in your 'conftest.py' file
+REQUIRED: Dict[str, Any] = {}
+
+# The following parameters are optional and can be used to customize your tests
+# You can set their values in your 'conftest.py' file if you want to use them
+OPTIONAL: Dict[str, Any] = {
+
+    # use this parameter if you want Speculos to emulate other applications being installed on the device
+    # example: {"bitcoin": "Bitcoin", "ethereum": "Ethereum"}
```

Comment:
Can you explain what should be use for the key and the value and what are both used for?

Reply:
Done